### ISSUE TITLE
Create a simple Java example in the documentation

### ISSUE THREAD

**Issue by pmcb55** [graphframes/graphframes#69]
The documentation currently only provides Scala and Python examples, but I can't find any Java examples online.

For instance these two links work fine:
http://go.databricks.com/hubfs/notebooks/3-GraphFrames-User-Guide-scala.html
http://go.databricks.com/hubfs/notebooks/3-GraphFrames-User-Guide-python.html

But what I assume should be the Java examples, gives a 403 Forbidden response:
http://go.databricks.com/hubfs/notebooks/3-GraphFrames-User-Guide-java.html

**Comment by thunterdb** [comment]
Yes, the interface is usable by java, but there is no official documentation for java. Feel free to submit a pull request if you happen to write such a document.

**Comment by treena123** [comment]
I needed to implement GraphFrame for a java based project.  So, I thought to prepare a document for GraphFrame Implementation using java
[GraphFrame java documentation.pdf](https://github.com/graphframes/graphframes/files/471087/GraphFrame.java.documentation.pdf)

**Comment by mobiusinversion** [comment]
@treena123 very useful thank you, and thank you for fixing the annoying lack of documentation 👍

**Comment by felixcheung** [comment]
hmm, how is this created? is this something we could add to the project so it gets to be rebuilt for each release?

**Comment by treena123** [comment]
As I mentioned earlier, I needed to implement graphframe using java for one of my projects. So, I took a reference of python ,scala based graphframe documentaion and created this document.Yeah, you can add it to the project.

**Comment by rickyhai11** [comment]
Dear Folks, 

Is there any existing GraphFrame user guide document for Java developer ?
I am looking for an official document. Please let me know if this issue has been solved. Thanks

**Comment by mohibkhan-nedian** [comment]
> I needed to implement GraphFrame for a java based project. So, I thought to prepare a documentation
> for GraphFrame Implementation using java
> [GraphFrame java documentation.pdf](https://github.com/graphframes/graphframes/files/471087/GraphFrame.java.documentation.pdf)

@treena123 Can you please tell how did you add GraphFrame dependency? i could not find it in Maven central repository.

**Comment by ilya1245** [comment]
Java solution:
GraphFrame gf = GraphFrame.apply(verDF, edgeDF);

**Comment by eyala** [comment]
What is the issue here? Adding parts of the documentation pdf that @treena123 created to [quick-start.md](https://github.com/graphframes/graphframes/blob/master/docs/quick-start.md?plain=1)?
